### ISSUE TITLE
More music to choose from

### ISSUE THREAD

**Issue by Meorge** [LuisMayo/objection_engine#95]
In addition to the "Trial" and "Pursuit" tracks, I thought it might be nice to potentially have cross-examination and objection themes play. My branch [cross-examination-music](https://github.com/Meorge/objection_engine/tree/cross-examination-music) currently has:
- *Cross-Examination ~ Moderato*
- *Cross-Examination ~ Allegro*
- *Trial* (`trial.mp3`)

as "relaxed" tracks (that play at the beginning of a clip), and
- *Phoenix Wright ~ Objection!*
- *Pursuit ~ Cornered* (`press.mp3`)

as "tense" tracks that start playing after an Objection.

This is sample output from it, showcasing the different themes:

https://user-images.githubusercontent.com/9957987/201011340-2d062f61-31d9-4eca-bf9e-57fa4008d1a6.mp4

Unfortunately, right now it switches the "tense" track every time there's an Objection... that's (probably) something I'd need to fix, because otherwise in a heated conversation it might keep switching between tracks really quickly. Shouldn't be too hard of a fix, though - probably just has to do with reworking how the `if last_audio != new_audio` logic around line 550 of `anim.py` works a little bit.

**Comment by LuisMayo** [comment]
That could bring more variety. I understand your intention is to keep the same music flow as before, isn't it?

It starts on relaxed, it goes to pursuit/cornered/action music and there is no coming back then.  Or were you thinking about like a 2 layer objection system where there is a "cornered" track and a "more cornered" one? :P

I agree that the re-starting music issue should be fixed. As you suggested that line may be the culprit, it __shouldn't__ be too hard to do it

**Comment by Meorge** [comment]
> That could bring more variety. I understand your intention is to keep the same music flow as before, isn't it?
> 
> It starts on relaxed, it goes to pursuit/cornered/action music and there is no coming back then. Or were you thinking about like a 2 layer objection system where there is a "cornered" track and a "more cornered" one? :P

Yes, for now I was thinking it'd be best to keep the same music flow (a "relaxed" phase and then a "cornered" phase), but where the actual track played for each could be a random choice. It'd be really neat if we could figure out an algorithm for changing the music appropriately more, but it's something I've been thinking about for a while and I haven't been able to come up with a good method.

> I agree that the re-starting music issue should be fixed. As you suggested that line may be the culprit, it shouldn't be too hard to do it

I'll try to get that fixed today!
